typescript/phoenix-sdk/src/client.ts

```diff
@@ -167,6 +167,45 @@ export class Client {
     });
   }
 
+  static async createWithoutConfig(
+    connection: Connection,
+    marketAddresses: PublicKey[],
+    trader?: PublicKey
+  ): Promise<Client> {
+    const accounts = await connection.getMultipleAccountsInfo(
+      [...marketAddresses, SYSVAR_CLOCK_PUBKEY],
+      "confirmed"
+    );
+
+    const clockBuffer = accounts.pop()?.data;
+    if (clockBuffer === undefined) {
+      throw new Error("Unable to get clock");
+    }
+    const clock = deserializeClockData(clockBuffer);
+
+    const marketKeysToData: Array<[PublicKey, AccountInfo<Buffer>]> =
+      marketAddresses.map((marketAddress, index) => {
+        return [marketAddress, accounts[index] as AccountInfo<Buffer>];
+      });
+
+    const [markets, tokens] = Client.loadMarketsAndTokens(marketKeysToData, []);
```

Comment:
Should we add a comment here that notes the returned tokens will be empty and to use one of the other methods if that info is needed?

Reply:
I don't think the contents will be empty, but the metadata should be. There are a lot of really wacky abstraction violations throughout this class IMO :(

Reply:
Ah I read that wrong. Yeah looks like the metadata fields will be just be a slice of the mint key for name and the full mint key for symbol